### ISSUE TITLE
Field to deploy on 'node 11' on 'devnet'

### ISSUE THREAD

**Issue by Mahmoud-Emad** [threefoldtech/zos#1744]

**Comment by muhamadazmy** [comment]
Can you provide more information about this issue for example the size of the disk you are trying to reserve. I checked the node directly and it shows it only has 39 G of free ssd space.

I also would like to know how did u calculate "available" free space on this node. So i can tell if this matches the reality or not.

**Comment by muhamadazmy** [comment]
Okay so i checked both the explorer, and graph ql and found out that might to be the issue:
- The node has some deployed contracts that are not in `Created` state
- By checking out the state of those contracts, the seem to be in the `GracePeriod` state
- This grace period contract has 200GB reserved vdisk + 4 G to rootfs
- This adds up to the missing capcity

I am wondering if the explorer does not consider contracts that are in GracePeriod as used capacity. 

@OmarElawady can you confirm that assumption, If true explorer then need to take into account capacity used by grace periods contracts and this issue then need to be moved to the gridproxy.

**Comment by AhmedHanafy725** [comment]
@muhamadazmy yes the contracts with the state `Created` are only taken into consideration
https://github.com/threefoldtech/tfgridclient_proxy/blob/development/internal/explorer/db/postgres.go#L54
